middleware/allRoutes/handler.go

```diff
@@ -3,18 +3,17 @@ package allRoutes
 import (
 	"encoding/json"
 	"fmt"
-	"io"
-	"io/ioutil"
 	"net/http"
 	"regexp"
 
+	client "github.com/ONSdigital/dp-api-clients-go/zebedee"
 	"github.com/ONSdigital/dp-frontend-router/config"
 	"github.com/ONSdigital/go-ns/common"
-	"github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/log.go/log"
 )
 
 //Handler ...
```

Comment:
I know this was not you, but we should put valid comments on exported functions

Reply:
Also, I will create a constant for the ONS-Page-Type header, so that it's not buried in the function, and the logic is clearer.